### ISSUE TITLE
Vagrant No space left on device

### ISSUE THREAD

**Issue by minhvn** [devrandom/gitian-builder#192]
Please help, Im got message from vagrant :)
Thank you very much.
> STRIPPROG="/home/ubuntu/wrapped/i686-w64-mingw32-strip" /bin/bash /home/ubuntu/build/pln/distsrc-i686-w64-min                                                gw32/build-aux/install-sh -c -s ./src/plnd.exe ./release
> cp: error writing ‘./release/_inst.4604_’: No space left on device
> cp: failed to extend ‘./release/_inst.4604_’: No space left on device
>

**Comment by minhvn** [comment]
I'm need resize vdi vmbox.
Thank all,

**Comment by minhvn** [comment]
@devrandom still error, I increase my vbox, how to change size in Vagarnt size? Thank you.
> /usr/bin/i686-w64-mingw32-strip:./release/stNc17w0[.text]: No space left on device

`/dev/sda1        87G   30G   53G  37% /
udev             10M     0   10M   0% /dev
tmpfs          1012M  8.5M 1004M   1% /run
tmpfs           2.5G     0  2.5G   0% /dev/shm
tmpfs           5.0M     0  5.0M   0% /run/lock
tmpfs           2.5G     0  2.5G   0% /sys/fs/cgroup
tmpfs           506M     0  506M   0% /run/user/1000
`

**Comment by devrandom** [comment]
Likely needs a change to `Vagrantfile`.

**Comment by minhvn** [comment]
Yes, where line I can change on that? I add config.disksize.size = '60GB' ? but it not affect.
And where I can remove old file on that?
Thank @devrandom . Please help me.

**Comment by minhvn** [comment]
Hic, change other mdi, still got error :(

**Comment by minhvn** [comment]
VBox show ext4-fs loop 7 couldn't mount as ext2 due to feautre incompatiblietes when build for windows :(
